libnavigator/src/test/java/com/mapbox/navigation/navigator/NavigatorMapperTest.kt

```diff
@@ -440,7 +443,10 @@ class NavigatorMapperTest {
 
     private val tunnelEntrancePassiveManeuver = createPassiveManeuver(
         hasLength = true,
-        type = PassiveManeuverType.KTUNNEL_ENTRANCE
+        type = PassiveManeuverType.KTUNNEL_ENTRANCE,
+        tunnelInfo = PassiveManeuverTunnelInfo(
+            "Fake Tunnel"
```

Comment:
```suggestion
            "St. Gotthard"
```

nit: real tunnel for successful tests, fake tunnel to test failures

Reply:
It's funny I had `Ted Williams Tunnel` in the first place. Will bring it back 😅